reports/report_builder.py

```diff
@@ -476,7 +476,7 @@ def ws_01(self, ws):
         self.tabulate(ws, counts_list[1], DM_MEDIA_TYPES, self.regions, row_perc=True, c=c, write_row_headings=False)
         c = ws.dim_colmax + 2
 
-        self.tabulate_historical(ws, '01', TM_MEDIA_TYPES, self.regions, c=c)
+        self.tabulate_historical(ws, '01', TM_MEDIA_TYPES+DM_MEDIA_TYPES, self.regions, c=c)
```

Comment:
Instead of `TM_MEDIA_TYPES+DM_MEDIA_TYPES`, why are we not using [MEDIA_TYPES](https://github.com/CodeForAfrica/gmmp/blob/6c154d7db5af9213cd5b23606bc3eda094a9f4a4/forms/modelutils.py#L261-L267)?

Reply:
I missed that. lemme update it.